ctx/ctx.go

```diff
@@ -40,6 +42,7 @@ const (
 	UrlRewriteTarget
 	TransformedRequestMethod
 	Definition
+	OASDefinition
```

Comment:
Whilst this might not be a breaking change, I think it is a good practise to add new constants to iota to the end of the list. If this was ever to be persisted, it would be have been a breaking change

Reply:
Yes, if it was a persistent thing, I'd respect the order but here in terms of readability, `Definition` and `OASDefinition` should follow each other. Wdyt?

Reply:
Iota is a numerical value, so previously RequestStatus would have been 21, now it would be 22; if we store it in databases, it would be a breaking change (and we can't really prevent using this value wrongly).

If the type was not iota, but rather `struct{}`, then we wouldn't have this possibility of conflict, but this too is a breaking change. Plugins use these ctx values, and may possibly store them, so end of the list sounds like the best idea for iota values.